### PR TITLE
EVG-1496 Modify TaskQueueDistros resolver to also return HostCount

### PR DESCRIPTION
**Pull request by minnakt**
[EVG-1496](https://jira.mongodb.org/browse/EVG-1496)

### Description 
This PR modifies the `TaskQueueDistros` resolver so that it returns the number of hosts associated with each distro. Previously, this function would only return the distro `ID` and `QueueCount` (the number of tasks on the queue).

### Testing 
- Modified the tests in `gql/tests/taskQueueDistros`

### CODE REVIEW

File: graphql/resolvers.go [hunk 1]
```diff
@@ -2205,16 +2205,21 @@ func (r *queryResolver) TaskQueueDistros(ctx context.Context) ([]*TaskQueueDistr
 	distros := []*TaskQueueDistro{}
 
 	for _, distro := range queues {
+		numHosts, err := host.CountRunningHosts(distro.Distro)
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting associated hosts: %v", err.Error()))
```
Comment (SupaJoon) [inline graphql/resolvers.go:2210]:
```suggestion
			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting associated hosts: %s", err.Error()))
```

**Review by SupaJoon — APPROVED** [review]

File: graphql/schema.graphql [hunk 2]
```diff
@@ -672,7 +672,8 @@ type TaskQueueItem {
 
 type TaskQueueDistro {
   id: ID!
-  queueCount: Int!
```
Comment (SupaJoon) [inline graphql/schema.graphql:675]: we should keep this field to retain backwards compatability

**Review by SupaJoon — CHANGES_REQUESTED** [review]

File: graphql/schema.graphql (hunk 2 above)
Comment (khelif96) [reply graphql/schema.graphql:675]: This is actually safe to remove it isn't used anywhere on the front end.

**Review by khelif96 — APPROVED** [review]

File: graphql/schema.graphql (hunk 2 above)
Comment (SupaJoon) [reply graphql/schema.graphql:675]: queueCount is queried here: https://github.com/evergreen-ci/spruce/blob/main/src/pages/TaskQueue.tsx#L32-L35
as part of this page: https://spruce.mongodb.com/task-queue/rhel71-power8-small

**Review by SupaJoon — APPROVED** [review]

**Pull request merged**